### PR TITLE
KM-798 logo & hero section changed

### PR DESCRIPTION
**Pull request by quinnnaira**

### CODE REVIEW

**Comment by ghost** [comment]
## :x: ip-nca deploy preview failed

 Your build failed. [View the build logs](https://gatsbyjs.com/dashboard/32488017-fa49-4557-966e-d7557a4eab4c/sites/67917ec1-d263-4f6f-982a-1d882a031ce5/builds/381b2c4f-03e7-4d89-b270-64b6ebfb5656/details).

**Errors**
<pre>PageCreator: Tried to create pages from the collection builder.
Unfortunately, the query came back empty. There may be an error in your query:

Cannot query field "allContentfulPage" on type "Query". Did you mean "allContentfulAsset", "allContentfulEntry", "allContentfulBlogPost", or "contentfulAsset"?</pre>
<pre>PageCreator: Tried to create pages from the collection builder.
Unfortunately, the query came back empty. There may be an error in your query:

Cannot query field "allContentfulPost" on type "Query". Did you mean "allContentfulAsset", "allContentfulBlogPost", "allContentfulEntry", "contentfulAsset", or "contentfulBlogPost"?</pre>
<pre>PageCreator: Tried to create pages from the collection builder.
Unfortunately, the query came back empty. There may be an error in your query:

Cannot query field "allContentfulProduct" on type "Query". Did you mean "allContentfulAsset", "allContentfulBlogPost", "allContentfulEntry", "contentfulAsset", or "contentfulBlogPost"?</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "allContentfulPost" on type "Query".

If you don't expect "allContentfulPost" to exist on the type "Query" it is most likely a typo.
However, if you expect "allContentfulPost" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "allContentfulPost" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "allContentfulProduct" on type "Query".

If you don't expect "allContentfulProduct" to exist on the type "Query" it is most likely a typo.
However, if you expect "allContentfulProduct" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "allContentfulProduct" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "allContentfulProduct" on type "Query".

If you don't expect "allContentfulProduct" to exist on the type "Query" it is most likely a typo.
However, if you expect "allContentfulProduct" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "allContentfulProduct" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "allContentfulPost" on type "Query".

If you don't expect "allContentfulPost" to exist on the type "Query" it is most likely a typo.
However, if you expect "allContentfulPost" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "allContentfulPost" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "contentfulPage" on type "Query".

If you don't expect "contentfulPage" to exist on the type "Query" it is most likely a typo.
However, if you expect "contentfulPage" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "contentfulPage" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "contentfulPost" on type "Query".

If you don't expect "contentfulPost" to exist on the type "Query" it is most likely a typo.
However, if you expect "contentfulPost" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "contentfulPost" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>
<pre>There was an error in your GraphQL query:

Cannot query field "contentfulProduct" on type "Query".

If you don't expect "contentfulProduct" to exist on the type "Query" it is most likely a typo.
However, if you expect "contentfulProduct" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "contentfulProduct" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add at least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "Query":
https://www.gatsbyjs.com/docs/reference/graphql-data-layer/schema-customization#creating-type-definitions</pre>

**Comment by konfirmed** [comment]
@quinnnaira this did not build at all.

File: contentful-import-error-log-SPACE_ID-2023-02-23T16-13-44.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[
```
Comment (konfirmed) [inline contentful-import-error-log-SPACE_ID-2023-02-23T16-13-44.json:1]: why is this pushed? @quinnnaira

**Review by konfirmed — COMMENTED** [review]
@quinnnaira this build failed completely

**Pull request merged**